bedrock/settings/static_media.py

```diff
@@ -1248,6 +1255,12 @@
         ),
         'output_filename': 'js/firefox_firstrun-bundle.js',
     },
+    'firefox_new_firstrun': {
+        'source_filenames': (
+            'js/firefox/firstrun/new-firstrun.js',
```

Comment:
will do, btw, it was for those two files that I was including `firefox_accounts`